decred_wallet/Extensions/Extensions.swift

```diff
@@ -69,3 +69,11 @@ extension Date {
         self = Date(timeIntervalSince1970: TimeInterval(milliseconds) / 1000)
     }
 }
+
+extension UIRefreshControl {
+    func programaticallyBeginRefreshing(in tableView: UITableView) {
+        beginRefreshing()
+        let offsetPoint = CGPoint.init(x: 0, y: -frame.size.height)
```

Comment:
Please add a comment as to why this is necessary.

Reply:
This is for displaying the loading indicator without the user swiping down the tableview